WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -806,7 +787,7 @@ private void trackModerationFromNotification(final CommentStatus newStatus) {
                 AnalyticsTracker.track(Stat.NOTIFICATION_APPROVED);
                 break;
             case UNAPPROVED:
-                AnalyticsTracker.track(Stat.NOTIFICATION_UNAPPROVED);
+                AnalyticsTracker.track(Stat.NOTIFICATION_APPROVED);
```

Comment:
I don't think you meant to change this to `NOTIFICATION_APPROVED`, unless I'm missing something.